patches/metainfo.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/data/metainfo.xml b/data/metainfo.xml
+new file mode 100644
+index 00000000..6f37c6fe
+--- /dev/null
++++ b/data/metainfo.xml
```

Comment:
Would be nice to upstream this

Reply:
metainfo.xml was recently upstreamed ([commit](https://github.com/narunlifescience/AlphaPlot/commit/70c42625fbe3a3ca81875b27d1ca93efb2955d9d)).

In the next release, this patch will be deleted.